src/main/webapp/components/sources-info-local/sources-info-local.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react'
+
+import { useQuery } from '@apollo/react-hooks'
+import gql from 'graphql-tag'
+
+import Card from '@material-ui/core/Card'
+import CardContent from '@material-ui/core/CardContent'
+import Divider from '@material-ui/core/Divider'
+import LinearProgress from '@material-ui/core/LinearProgress'
+import StorageIcon from '@material-ui/icons/Storage'
+import List from '@material-ui/core/List'
+import ListItem from '@material-ui/core/ListItem'
+import ListItemIcon from '@material-ui/core/ListItemIcon'
+import ListItemText from '@material-ui/core/ListItemText'
+import Typography from '@material-ui/core/Typography'
+import OnlineIcon from '@material-ui/icons/CloudDoneOutlined'
+import OfflineIcon from '@material-ui/icons/OfflineBoltOutlined'
+import { SnackbarRetry } from '../network-retry'
+import { SourcesInfo } from '../sources-info/sources-info'
+
+const sources = gql`
```

Comment:
You can combine the `sources` and `facetedQuery` gql tags into one which will remove the need for the isSourceReady logic as both queries will share the same loading and error state:
```
const query = gql`
  query SourcesInfoLocal($attribute: String!) {
    facet(attribute:$attribute) {
      value
      count
    }
    sources {
      isAvailable
      sourceId
      local
    }
    
  }
`
```